vertical-pod-autoscaler/pkg/utils/metrics/recommender/recommender.go

```diff
@@ -134,11 +135,13 @@ func (oc *ObjectCounter) Add(vpa *model.Vpa) {
 	if vpa.IsV1Beta1API {
 		api = v1beta1
 	}
+
 	key := objectCounterKey{
-		mode:        mode,
-		has:         vpa.HasRecommendation(),
-		apiVersion:  api,
-		matchesPods: vpa.HasMatchedPods(),
+		mode:              mode,
+		has:               vpa.HasRecommendation(),
+		apiVersion:        api,
+		matchesPods:       vpa.HasMatchedPods(),
+		unsupportedConfig: vpa.Conditions.ConditionActive(vpa_types.ConfigUnsupported),
```

Comment:
I think you need to modify also the NewObjectCounter function.

Reply:
Done.